test/scoreColorTest.java

```diff
@@ -30,6 +30,16 @@ private static Color blinkingColor(String color) {
             Color gray = new Color(RGB, RGB, RGB);
             return gray;
         }
+        if (color == "RANDOM") {
```

Comment:
After implementing the feature, I noticed that it was working similar to the ```color == "HIGH_SCORES"```.

However, it has the advantage of being able to **specify colors**, which is advantageous for future development. Please refer to it!

Thank you.